### PR TITLE
Bumping build number - PugiXML conda break

### PR DESCRIPTION
**Pull request by hansenms**
This PR bumps the build number of the conda package. 

The conda-feedstock package for PugiXML was actually a static library, which we depended on, but they decided to move to a shared library, but they did it without bumping the version number of the package. This will pull in this change and create a new build version for us. It is to reduce the downstream breakage in various packages where build strings are not pinned. 

See: https://github.com/conda-forge/pugixml-feedstock/pull/22

There will be a subsequent package to pull in a later version of PugiXML and rev the version of ISMRMRD.

### CODE REVIEW

**Review by johnstairs — APPROVED** [review]

File: include/ismrmrd/xml.h [hunk 1]
```diff
@@ -31,164 +32,157 @@
  * @{
  */
 
-namespace ISMRMRD
-{
+namespace ISMRMRD {
```
Comment (johnstairs) [inline include/ismrmrd/xml.h:35]: It looks like this file was reformatted. Maybe do that in a separate PR?

Comment (hansenms) [reply include/ismrmrd/xml.h]: Yes, reverted that.
